### PR TITLE
Fix for email field

### PR DESCRIPTION
**Pull request by Mte90**
Subtype wasn't used so the email field was rendered as `text`

### CODE REVIEW

File: dynamic_forms/utils.py [hunk 1]
```diff
@@ -92,6 +92,8 @@ def process_field_from_json(field_json):
     if not isinstance(field_json, dict):
         raise TypeError("Each field JSON must be a dictionary")
     field_type = field_json['type']
+    if field_type == 'text':
+        field_type = field_json['subtype']
```
Comment (askvortsov1) [inline dynamic_forms/utils.py:96]:
```suggestion
        field_type = field_json.get('subtype', 'text')
```

Wouldn't this be a bit more robust?

File: dynamic_forms/widgets.py [hunk 2]
```diff
@@ -44,3 +44,6 @@ def id_for_label(self, id):
 
     def get_context(self):
         return {'name': ''}
+    
+    def value_from_datadict(self, data, files, name):
+        return data.get(name)
```
Comment (askvortsov1) [inline dynamic_forms/widgets.py:49]: Why is this change necessary?

**Review by askvortsov1 — CHANGES_REQUESTED** [review]

File: dynamic_forms/widgets.py (hunk 2 above)
Comment (Mte90) [reply dynamic_forms/widgets.py:49]: This was required as it was missing that method to create this fake model and Django was generating an error.
